pkg/networkpolicy/multinetegressrule_test.go

```diff
@@ -84,6 +125,72 @@ func TestEgressWithPeerPodSelector(t *testing.T) {
 	assert.Len(t, builder.definition.To, 1)
 }
 
+func TestEgressWithPeerNamespaceSelector(t *testing.T) {
+	testCases := []struct {
+		namespaceSelector metav1.LabelSelector
+		expectedError     string
+	}{
+		{
+			namespaceSelector: metav1.LabelSelector{MatchLabels: map[string]string{"app": "nginx"}},
+			expectedError:     "",
+		},
+		{
+			namespaceSelector: metav1.LabelSelector{MatchExpressions: []metav1.LabelSelectorRequirement{
```

Comment:
In order to be consistent please add expectedError to text case.

Reply:
updated